qa/tasks/cephfs/test_volumes.py

```diff
@@ -2869,12 +2869,39 @@ def test_subvolume_info(self):
 
         self.assertEqual(subvol_info["earmark"], earmark)
         
+        self.assertEqual(subvol_info['source']['volume'], 'N/A')
+        self.assertEqual(subvol_info['source']['subvolume'], 'N/A')
+        self.assertEqual(subvol_info['source']['snapshot'], 'N/A')
+        self.assertEqual(subvol_info['source']['group'], 'N/A')
+        self.assertEqual(subvol_info['source']['subvolume_state'], 'N/A')
+
         # remove subvolumes
         self._fs_cmd("subvolume", "rm", self.volname, subvolume)
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_info_with_custom_group(self):
```

Comment:
Please add the test case of fetching the clone subvolume info when the

1. subvolume is deleted
2. subvolume is deleted and recreated with same name
2. subvolume is deleted with retained snapshot

Reply:
@kotreshhr Note that these tests aren't applicable anymore due to our discussion (here's the summary of it: https://github.com/ceph/ceph/pull/63214#discussion_r2120267079) and therefore they'll be removed in upcoming update.